### ISSUE TITLE
Train with own dataset

### ISSUE THREAD

**Issue by AndreaPisoni** [mitmul/chainer-faster-rcnn#11]
Hi! Thanks a lot for doing this! How would i go about training the model on my own dataset?

**Comment by apple2373** [comment]
Hi, I am trying to do this too. Did you make it? 

I guess you need to make a dataset class like this: https://github.com/apple2373/chainer-faster-rcnn/blob/master/lib/dataset_custom.py

The data can be stored in the json like this: 
[
    {
        "bndbox": [
            0.0, 
            57.0, 
            368.0, 
            457.0
        ], 
        "difficult": 0, 
        "filename": "000494.jpg", 
        "name": "car", 
        "pose": "unspecified", 
        "truncated": 0
    }, 
    ....

I guess we only need "filename", "name", and "bndbox" for training.

**Comment by kirk86** [comment]
@apple2373 have you tested it. What does "difficult":0, "truncated":0 and "pose": "unspecified" mean?

**Comment by apple2373** [comment]
Nope, I am using YOLO now.

**Comment by kirk86** [comment]
@apple2373 does it give you good results?

**Comment by apple2373** [comment]
Yes, at least I was able to solve my own problem.  I am using YOLO 2 with darknet and darkflow (tensorflow ported version), but I am also aware of chained ported version. YOLO2 is much simpler and faster than faster R-CNN. I haven't trained on VOC or COCO but YOLO2 is reported to have better mAP than faster R-CNN.

**Comment by kirk86** [comment]
@apple2373 thanks for the update.
